docs/getting_started/authentication.rst

```diff
@@ -41,18 +41,18 @@ Password Flow
 **Password Flow** is the simplest type of authentication flow to work with
 because no callback process is involved in obtaining an ``access_token``.
 
-While **password flow** applications do not involve a redirect uri, Reddit
+While **password flow** applications do not involve a redirect URI, Reddit
```

Comment:
Shouldn't this be URL?

Reply:
Nope — URI is Uniform Resource Identifier. A URL is a type of URI. See https://en.wikipedia.org/wiki/Uniform_Resource_Identifier.

Reply:
Thanks @jarhill0